src/main/java/de/metas/ui/web/window/model/DocumentCollection.java

```diff
@@ -231,7 +228,7 @@ private Document getOrLoadDocument(@NonNull final DocumentKey documentKey)
 	{
 		final DocumentKey rootDocumentKey = DocumentKey.ofRootDocumentPath(documentPath.getRootDocumentPath());
 
-		try (final IAutoCloseable readLock = getOrLoadDocument(rootDocumentKey).lockForReading())
+		try (@SuppressWarnings("unused") final IAutoCloseable readLock = getOrLoadDocument(rootDocumentKey).lockForReading())
```

Comment:
do we really need this? who complains when we don't have this `@SuppressWarnings` in a try-with-ressource?

Reply:
Intellij shows me a warning that we should rename `readLock` to `ignored` since it is not used anywhere
![image](https://user-images.githubusercontent.com/4482210/79419335-4b7a2a80-7fbf-11ea-8e7c-ce4b05d52478.png)

I believe `readLock` is better than ignore to show the purpose of the variable, so i suppressed that warning.

Reply:
MM discussion: tobi believes `ignored` is better.